src/components/basic/Dialog/index.tsx

```diff
@@ -46,13 +46,14 @@ export const Dialog = ({
   ...props
 }: DialogProps & AddtionalDialogProps) => {
   const theme = useTheme()
+  const notWeb = useMediaQuery('(max-width:1023px)')
 
   const radius =
     modalBorderRadius && modalBorderRadius !== 0 ? modalBorderRadius : 20
 
   const fullScreenWidth = `calc(100vw - ${theme.spacing(8)})`
-
-  const defaultWidth = maxWidth ? 'auto' : MODAL_DEFAULT_WIDTH
+  // eslint-disable-next-line
+  const defaultWidth = maxWidth || notWeb ? 'auto' : MODAL_DEFAULT_WIDTH
```

Comment:
replace || with ?? to remove exception above

Reply:
Here || is required. when maxWidth or notWeb is present we have to use auto

Reply:
okay, then explicit check for possible negative values is still better than the linter exception:

`maxWidth === false || maxWidth === undefined || notWeb`